tts/test/test_unit_synthesizer.py

```diff
@@ -142,6 +142,200 @@ def test_cli_engine_dispatching_3(self, speech_synthesizer_class_mock):
             speech_synthesizer_class_mock.assert_called_with(engine='POLLY_SERVICE', polly_service_name='apolly')
             self.assertGreater(speech_synthesizer_class_mock.return_value.start.call_count, 0)
 
+    def test_repeated_synthesis(self):
+        from tts.db import DB
+        from tts.synthesizer import SpeechSynthesizer
+        from tts.srv import SynthesizerRequest
+        import uuid
+
+        db = DB()
+        init_num_files = db.get_num_files()
```

Comment:
Should you have an assert that `init_num_files == 0`? So later on, your test should be `db.get_num_files() == 1` instead of `db.get_num_files() == init_num_files + 1`.

Reply:
This may need a deletion of `/tmp/polly.db` at the end of each test case.

What is the right way to delete a database and all the audio files associated with it?